dask/dataframe/core.py

```diff
@@ -336,6 +335,25 @@ def quantiles(self, q):
         """
         return quantiles(self, q)
 
+    def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
+                 label=None, convention='start', kind=None, loffset=None,
+                 limit=None, base=0):
+        first, last = self.divisions[0], self.divisions[-1]
+        new_divisions = list(pd.date_range(start=first, end=last, freq=rule))
```

Comment:
I'm sorry, after rereading your question and my answer I realize that my answer really should have been the following

    Yes.